src/screens/NewTerm.js

```diff
@@ -55,7 +68,9 @@ export default ({
   // find the selected category
   const selectCategory = Text => {
```

Comment:
naming convention broken `Text` this should be renamed to something like categoryName